src/preview.js

```diff
@@ -12,11 +12,34 @@ const getPreviewHeader = ( lang, imageUrl = '' ) =>{
 	`.trim()
 	},
 
-	renderPreview = ( lang, data, isTouch ) => {
-		const imageUrl = data.imgUrl
+	getPreviewBody = ( type, message, cta ) => {
+		return `
+			<div class="wikipediapreview-body">
+				<div class="wikipediapreview-body-template ${type}">
```

Comment:
the type here `error` `disambiguation` `offline` can be overwritten by client website if they have the same class name with `!important`.

Reply:
Good call!

Reply:
I'm not sure what the "-template" part means. Why do we have `div.body-template` inside `div.body`?

Reply:
I think "-template" made more sense when we were including that CSS through `extend`.

What about replacing "-template" with "-state"?... or something along those lines

Reply:
Do we need 2 container levels? Can we have `<div class="wp-body wp-body-${type}">`?

Reply:
Actually you're right, check out https://github.com/wikimedia/wikipedia-preview/pull/81/commits/6d67140d5d2b7555694a7bd6812fce305b2b46f7. It abstracts it to only one container and now I think we can make a more appropriate use of `extend` with that `.wikipediapreview-body-non-regular`, similiar to what we're doing with `.license` in gallery.less (which by the way we should take care of [that TODO](https://github.com/wikimedia/wikipedia-preview/blob/master/style/gallery.less#L3) in either [T262371](https://phabricator.wikimedia.org/T262371) or [T262373](https://phabricator.wikimedia.org/T262373))